inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/listener/source/AbstractSourceOperateListener.java

```diff
@@ -99,6 +99,10 @@ protected void operateStreamSources(String groupId, String streamId, String oper
      */
     @SneakyThrows
     public boolean checkIfOp(StreamSource streamSource, List<StreamSource> unOperatedSources) {
+        // template source task need not be operated
```

Comment:
Where are the sub-sources operated?

Reply:
Also here.  The template task is filtered.
Comments added.